lib/Doctrine/DBAL/Query/Expression/ExpressionBuilder.php

```diff
@@ -68,7 +71,7 @@ public function or(...$expressions) : CompositeExpression
      */
     public function andX($x = null)
     {
-        return $this->and(...func_get_args());
+        return new CompositeExpression(CompositeExpression::TYPE_AND, func_get_args());
```

Comment:
I had to revert this to its original value; we can't delegate to `and()`, as `andX()` accepts an empty parameter list.